<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>59</y>
     <width>261</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>261</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true">QWidegt{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="FriendList">
        <widget class="QTreeWidget" name="FriendTree">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>290</y>
           <width>261</width>
           <height>261</height>
          </rect>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>261</width>
           <height>91</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget_4{
	background-color: rgb(255, 255, 255);
	border-bottom: none;
	border-top:none;
	border-left:none;
}</string>
         </property>
         <widget class="QPushButton" name="AddFriend_Room">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>211</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton::menu-indicator{
	image:none;
}</string>
          </property>
          <property name="text">
           <string>添加好友</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>269</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget_5{
	background-color: rgb(255, 255, 255);
	border-bottom: none;
	border-top:none;
	border-left:none;
}</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>好友</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget_6{
	background-color: rgb(255, 255, 255);
	border-bottom: none;
	border-top:none;
	border-left:none;
}</string>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>群聊</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="GroupList" native="true">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>109</y>
           <width>261</width>
           <height>161</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#GroupList{
	background-color: rgb(255, 255, 255);
	border-bottom: none;
	border-top:none;
	border-left:none;
}</string>
         </property>
         <widget class="QTreeWidget" name="GroupTree">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>261</width>
            <height>192</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>751</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
}</string>
   </property>
   <widget class="QPushButton" name="FriendButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resource.qrc">
      <normaloff>:/images/friend_icon.png</normaloff>:/images/friend_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="UserJidEdit">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>59</y>
     <width>491</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>491</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
